src/DynamoPackages/PackageDirectoryBuilder.cs

```diff
@@ -92,6 +93,18 @@ private void RemoveDyfFiles(IEnumerable<string> filePaths, IDirectoryInfo dyfDir
             }
         }
 
+        private void RemoveOtherFiles(IEnumerable<string> filePaths, IDirectoryInfo dir)
+        {
+            // Remove all files which are not listed in the files list
+            Directory.GetFiles(dir.FullName, "*", SearchOption.AllDirectories)
+                .Where(x => !x.EndsWith("pkg.json") && !filePaths.Contains(x)).ToList()
```

Comment:
Take a look at the `IFileSystem` type. Please use it for all file system operations. You may need to extend it. It makes it easy to write tests for packages that do not mutate the file system.